packages/components/src/components/divider/divider.stories.ts

```diff
@@ -1,149 +1,67 @@
 import '../../solid-components';
 import { html } from 'lit';
-import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
-const { args, argTypes, parameters } = storybookDefaults('sd-divider');
-const { generateTemplate } = storybookTemplate('sd-divider');
+import { storybookDefaults, storybookHelpers } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-divider');
+const { overrideArgs } = storybookHelpers('sd-divider');
+
+/**
+ * Used to separate content or sections from each other and make the content easier to read for the user.
+ */
 
 export default {
   title: 'Components/sd-divider',
+  tags: ['!dev'],
   component: 'sd-divider',
-  args,
-
+  args: overrideArgs([{ type: 'slot', name: 'default', value: 'Default Slot' }]),
   argTypes,
-
-  parameters: { ...parameters },
-  decorators: [
-    (story: () => typeof html) => html`
-      <style>
-        sd-divider[orientation='vertical'] {
-          height: 120px;
-        }
-
-        sd-divider[orientation='horizontal'] {
-          width: 120px;
-        }
-
-        .size-sm sd-divider[orientation='horizontal'] {
-          width: 50px;
-        }
-
-        .size-sm sd-divider[orientation='vertical'] {
-          height: 50px;
-        }
-
-        .size-l sd-divider[orientation='horizontal'] {
-          width: 150px;
-        }
-
-        .size-l sd-divider[orientation='vertical'] {
-          height: 150px;
-        }
-      </style>
-      ${story()}
-    `
-  ]
+  parameters: { ...parameters }
 };
-/**
- * This shows sd-divider in its default state.
- */
 
 export const Default = {
-  render: (args: any) => {
-    return generateTemplate({ args });
-  }
+  name: 'Default',
+  render: () => html`<sd-divider orientation="horizontal" class="w-[120px]"></sd-divider>`
 };
```

Comment:
@auroraVasconcelos Please make dynamic again. Do it directly here in the branch. Please comment, when done.